ERROR error building site: render: [en v1.0.0 guest] failed to render pages

我尝试基于这个模板build自己的博客，但是有下面的报错

```js
Start building sites … 
[16](https://github.com/qstargazer/qstargazer.github.io/actions/runs/23367495707/job/67984315642#step:13:17)
hugo v0.158.0-f41be7959a44108641f1e081adf5c4be7fc1bb63+extended linux/amd64 BuildDate=2026-03-16T17:42:04Z VendorInfo=gohugoio
[17](https://github.com/qstargazer/qstargazer.github.io/actions/runs/23367495707/job/67984315642#step:13:18)

[18](https://github.com/qstargazer/qstargazer.github.io/actions/runs/23367495707/job/67984315642#step:13:19)
WARN  Search page not found. Create a page with layout: search.
[19](https://github.com/qstargazer/qstargazer.github.io/actions/runs/23367495707/job/67984315642#step:13:20)
WARN  Archives page not found. Create a page with layout: archives.
[20](https://github.com/qstargazer/qstargazer.github.io/actions/runs/23367495707/job/67984315642#step:13:21)
WARN  Taxonomy categories not found
[21](https://github.com/qstargazer/qstargazer.github.io/actions/runs/23367495707/job/67984315642#step:13:22)
ERROR error building site: render: [en v1.0.0 guest] failed to render pages: render of "/" failed: "/home/runner/work/_temp/hugo_cache/modules/filecache/modules/pkg/mod/github.com/!cai!jimmy/hugo-theme-stack/v4@v4.0.0-beta.14/layouts/home.html:15:7": execute of template failed: template: home.html:15:7: executing "right-sidebar" at <partial "sidebar/right.html" (dict "Context" . "Scope" "homepage")>: error calling partial: "/home/runner/work/_temp/hugo_cache/modules/filecache/modules/pkg/mod/github.com/!cai!jimmy/hugo-theme-stack/v4@v4.0.0-beta.14/layouts/_partials/sidebar/right.html:10:19": execute of template failed: template: _partials/sidebar/right.html:10:19: executing "_partials/sidebar/right.html" at <partial (printf "widget/%s" .type) (dict "Context" $context "Params" $params)>: error calling partial: "/home/runner/work/_temp/hugo_cache/modules/filecache/modules/pkg/mod/github.com/!cai!jimmy/hugo-theme-stack/v4@v4.0.0-beta.14/layouts/_partials/widget/categories.html:2:3": execute of template failed: template: _partials/widget/categories.html:2:3: executing "_partials/widget/categories.html" at <partial "widget/taxonomy" (dict "Context" .Context "Params" $params)>: error calling partial: "/home/runner/work/_temp/hugo_cache/modules/filecache/modules/pkg/mod/github.com/!cai!jimmy/hugo-theme-stack/v4@v4.0.0-beta.14/layouts/_partials/widget/taxonomy.html:24:37": execute of template failed: template: _partials/widget/taxonomy.html:24:37: executing "_partials/widget/taxonomy.html" at <$taxonomyPage.RelPermalink>: nil pointer evaluating page.Page.RelPermalink
[22](https://github.com/qstargazer/qstargazer.github.io/actions/runs/23367495707/job/67984315642#step:13:23)
Total in 470 ms
[23](https://github.com/qstargazer/qstargazer.github.io/actions/runs/23367495707/job/67984315642#step:13:24)
Error: Process completed with exit code 1.
```

具体的报错链接在[vault backup: 2026-03-21 08:07:18 · qstargazer/qstargazer.github.io@62eb9f3](https://github.com/qstargazer/qstargazer.github.io/actions/runs/23367495707/job/67984315642)，请帮忙看看具体是什么原因？如何修复？

--- a/layouts/_partials/widget/taxonomy.html
+++ b/layouts/_partials/widget/taxonomy.html
@@ -20,7 +20,7 @@
         {{ partial "helper/icon" $icon }}
     </div>
     <h2 class="widget-title section-title">
-        {{ if $showLink }}
+        {{ if and $showLink $taxonomyPage }}
             <a href="{{ $taxonomyPage.RelPermalink }}">
                 {{ $widgetTitle }}
             </a>

